broadinstitute/pearl
# AR-95 adding ourhealth participation page

This adds a new HtmlSectionTemplate "PARTICIPATION_DETAIL" to capture the step-by-step approach.  As usual, the html is rough-and-ready.  It's reasonably responsive, but probably needs more polish to get it fully mobile-ready, and also general pixel-polishing and font-weight-and-size refinement to get it closer to Erin's design.

TO TEST:
1. redeploy api-admin and api-participant servers
2. `./scripts/populate_portal.sh ourhealth`
3. confirm the "Participation" link appears in navbar
4. Click it, and confirm the participation page renders
![image](https://user-images.githubusercontent.com/2800795/220742159-bbe40b34-e8f9-4104-adbe-4862ad4df15d.png)
5. It should more-or-less match the Participant page at https://dspuxteam.invisionapp.com/console/Our-Health-cl6fbib5i1cpq01zec5qx922m/cleekk4n606b801a49nkmhbnb/play
